examples/qml_features/qml/pages/ExternCxxQtPage.qml

```diff
@@ -0,0 +1,69 @@
+// SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+import QtQuick.Window 2.12
+
+import com.kdab.cxx_qt.demo 1.0
+import com.kdab.cxx_qt.demo_cpp 1.0
+
+Page {
+    property int amount: 5
+
+    header: ToolBar {
+        RowLayout {
+            anchors.fill: parent
+
+            ToolButton {
+                text: qsTr("Trigger")
+
+                onClicked: rustExternCxxQt.triggerOnExternal(externalQObject, amountSpinBox.value)
+            }
+
+            Item {
+                Layout.fillWidth: true
+            }
+        }
+    }
+
+    ExternalQObject {
+        id: externalQObject
+    }
+
+    ExternalCxxQtHelper {
+        id: rustExternCxxQt
+    }
+
+    ColumnLayout {
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.verticalCenter: parent.verticalCenter
+
+        Label {
+            Layout.fillWidth: true
+            horizontalAlignment: Text.AlignHCenter
+            text: qsTr("Connecting to signals in external QObject can run closures as normal.")
```

Comment:
I think this explanation could be a bit more thorough.
```suggestion
            text: qsTr("CXX-Qt supports closures when connecting to signals.\n" +
                            "In this example the 'Trigger' button can be used to trigger a signal.\n" +
                            "A different object is captured by the invoked closure and this object increments the count.\n" +
                            "This way easy bindings between objects are possible with CXX-Qt.")
```
I think this page is otherwise a bit confusing when viewed without the accompanying count...

In general, I think `qml_features` suffers a lot from this.
The features are all shown on a technical level.
But the accompanying examples often don't really show anything exciting when viewing the QML GUI.

Also, this page doesn't really have much to do with `extern "C++Qt"`, but rather with the closures in signals...

Anyhow, I think a redo of some of the `qml_features` is in order anyhow.
Something for post-release maybe, so we can play more with the (more) stable API.

Reply:
done, and yes i think `qml_features` needs a proper overhaul at some point, another good task for newcomers / R&D time :thinking: